docs/partials/recording-gameplay/_data-zip-partial.js

```diff
@@ -11,6 +11,32 @@ Basic information about when this tick was captured within the context of the ov
 This includes the sequential tick number (which matches the name of the file), 
 the time in seconds since the start of the game, and the in-game timescale.
 
+### Key Frame Flags
+
+Key Frame flags indicate that important changes occurred to the game state since the previous tick was captured.
+Flags are represented as an array of strings in the \`keyFrame\` field.
+
+| Key Frame Flag              | Description                                                                                                                                                                   |
+|:----------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| FIRST_FRAME                 | Indicates that this tick is the first tick of the recording                                                                                                                   |
+| SCENE                       | Indicates that the visible scenes are different from those in the previous tick                                                                                               |
+| GAME_ELEMENT                | Indicates that a game element became visible to the camera or moved off-camera. This could be from spawning, de-spawning, character movement, camera rotation, etc.           |
+| GAME_ELEMENT_RENDERER_COUNT | Indicates that the number of renderers changed for a visible game element                                                                                                     |
+| UI_ELEMENT                  | Indicates that a UI element became visible to the camera or moved off-camera. This could be from spawning, de-spawning, UI component or canvas group visibility changes, etc. |
+| UI_PIXELHASH                | Indicates that the screen pixels of a UI component changed. This value is only relevant when using 3rd party UI systems like Coherent GameFace.                               |
+
```

Comment:
END_RECORDING is a new keyframetype recently added for the 'last segment' ...  this captures the state and any buffered inputs when the user stops recording